tests/test_entrypoint.py

```diff
@@ -33,3 +33,6 @@ def test_dotted_path_that_ends_with_yaml():
 
 def test_try_to_find_entry_point_type_with_none():
     assert try_to_find_entry_point_type(None) is None
+
+def test_find_entry_point_type_with_none():
```

Comment:
the test cases should be representative of what we wanna test, so try passing some value like 'some.value' and check that you get certain error type and error message, then pass something like 'file.yaml' and test that you get the other errors message 

here's an example:

```py
def test_recursion_depth():
    with pytest.raises(RuntimeError) as excinfo:

        def f():
            f()

        f()
    assert "maximum recursion" in str(excinfo.value)
```

source: https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions

Reply:
Something like this?

```
def test_find_entry_point_type_with_none():
    with pytest.raises(ValueError) as excinfo:
        find_entry_point_type('some.yaml')

    assert 'File does not exist' in str(excinfo.value)
```